### PR TITLE
chore: production build for website

### PR DESCRIPTION
**Pull request by tomaspalma**
This adds a nginx container and runs the adonisjs backend with pm2

### CODE REVIEW

File: deploy/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    container_name: enei-website
+    ports:
+      - "3333:3333"
+
+  website-nginx:
+    image: nginx:1-alpine
+    ports:
+      - "80:80"
+    volumes:
+      - ../website/nginx.conf:/etc/nginx/nginx.conf
```
Comment (limwa) [inline deploy/docker-compose.yaml:15]: I probably wasn't really clear. The docker-compose stays at the root of the repository.
All files needed for docker deployment stay in the `deploy` folder, including the dockerfile. The dockerignore must be at the depository root. The context for the build of all dockerfiles is at the root of the repository. For now, this doesn't make a lot of sense, but in the future, if we have multiple apps or packages in this repository, it's easier to build them and their local dependencies.

**Review by limwa — CHANGES_REQUESTED** [review]
Well done! Thank you so much, just a few organizational changes before we can merge!

File: deploy/docker-compose.yaml (hunk 1 above)
Comment (tomaspalma) [reply deploy/docker-compose.yaml]: I believe that is now fixed

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    container_name: enei-website
+    ports:
+      - "3333:3333"
+
+  website-nginx:
+    image: nginx:1-alpine
+    ports:
+      - "80:80"
+    volumes:
+      - ./deploy/nginx-website.conf:/etc/nginx/nginx.conf
```
Comment (limwa) [inline docker-compose.yaml:15]:
```suggestion
```
In production, nginx config doesn't need to be mounted as volumes. If we can create an image with this inside, it's perfect.

**Review by limwa — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (tomaspalma) [reply docker-compose.yaml]: Ok, it is now creating the image with it inside, instead of creating a volume

**Review by limwa — APPROVED** [review]
Thank you so much! Good work :rocket: 
Sorry for the fragmented reviews!

**Pull request merged**